akka-actor-typed/src/main/scala/akka/actor/typed/receptionist/Receptionist.scala

```diff
@@ -110,24 +129,124 @@ object Receptionist extends ExtensionId[Receptionist] {
   @InternalApi
   private[typed] abstract class InternalCommand extends AllCommands
 
+  /**
+   * Internal API
+   */
+  @InternalApi
+  private[akka] object MessageImpls {
```

Comment:
Maybe move them to `ReceptionistImpl` to get them out of view of API in the first place.

Reply:
@johanandren can you follow up on this comment - not important for the relase

Reply:
That's a separate file so then I can't have them sealed, but maybe that is nicer anyways.